packages/flutter/lib/src/rendering/view.dart

```diff
@@ -38,6 +38,18 @@ class ViewConfiguration {
     return Matrix4.diagonal3Values(devicePixelRatio, devicePixelRatio, 1.0);
   }
 
+  @override
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType)
+      return false;
+    return other is ViewConfiguration
+        && other.size == size
+        && other.devicePixelRatio == devicePixelRatio;
+  }
+
+  @override
+  int get hashCode => hashValues(size.hashCode, devicePixelRatio);
```

Comment:
IIRC `hashValues` already calls `.hashCode` on its arguments, so passing `size.hashCode` becomes `size.hashCode.hashCode`, which, while not wrong, is unnecessary.

Reply:
You are correct